src/io/flutter/FlutterUtils.java

```diff
@@ -94,7 +94,7 @@ public static boolean isInTestDir(DartFile file) {
     if (candidate == null) return false;
 
     final String relativePath = root.getRelativePath(candidate);
-    return relativePath != null && relativePath.startsWith("test/");
+    return relativePath != null && (relativePath.startsWith("test/") || relativePath.startsWith("example/test/"));
```

Comment:
We may want to tweak this check to:
- find the closest pubspec.yaml to `file`
- validate that `file` is within a `test` dir that's a sibling of the pubspec

Reply:
It looks like the previous version of `verifyFlutterTestFile()` was doing something very similar to that (looking for the closest containing pubspec.yaml making sure the relative path starts with `test/`), so I'm guess I'm a bit confused about the best fix.

Probably just best to figure out how  to support `test/`, `example/test/`, and `example/foo/test` in a relatively generic way...

Reply:
Absolutely.  I've been thinking some more about this and agree that this would be a more robust approach. 👍